packages/devtools/CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
-## 0.0.17-dev.2 - 2019-04-29
+## 0.0.18 - 2019-04-30
 * Fix release bug (0.0.17-dev.1 did not include build folder)
```

Comment:
wdyt about deleting this line? seems irrelevant now that the dev release headings have been removed.

Reply:
In Dart once we move to a alpha/stable we combine the interim devs to the new major release.